### PR TITLE
CLB - 22 Jul (even more)

### LINKED ISSUE #882: Journey to the Lost City

### LINKED ISSUE #887: Lae'zel's Acrobatics

### LINKED ISSUE #890: Loot Dispute

### LINKED ISSUE #892: Majestic Genesis

### LINKED ISSUE #898: Modify Memory

### PR DESCRIPTION
**Pull request by Northmoc**
- closes #882 
- closes #887 
- closes #890 
- closes #892 
- closes #898

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/loot_dispute.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Loot Dispute
+ManaCost:3 R
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigInitiative | TriggerDescription$ When CARDNAME enters the battlefield, you take the initiative and create a Treasure token.
+SVar:TrigInitiative:DB$ TakeInitiative | SubAbility$ DBTreasure
+SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac
+T:Mode$ AttackersDeclaredOneTarget | ValidAttackers$ Creature.YouCtrl | AttackedTarget$ Player.hasInitiative | Execute$ DBTreasure | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack the player who has the initiative, create a Treasure token.
+T:Mode$ DungeonCompleted | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you complete a dungeon, create a 5/5 red Dragon creature token with flying.
+SVar:TrigToken:DB$ Token | TokenScript$ r_5_5_dragon_flying
+DeckHas:Ability$LifeGain|Token|Sacrifice & Type$Artifact|Treasure|Dragon
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/loot_dispute.txt:10]: Where does LifeGain come from? Can't quite see the ability that causes it...

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/loot_dispute.txt:10]: heh, from copy/pasting 😅

File: forge-gui/res/cardsfolder/upcoming/modify_memory.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Name:Modify Memory
+ManaCost:4 U
+Types:Sorcery
+A:SP$ ExchangeControl | ValidTgts$ Creature | TgtPrompt$ Select target creature | TargetMin$ 2 | TargetMax$ 2 | TargetsWithDifferentControllers$ True | RememberExchanged$ True | SubAbility$ DBDraw | SpellDescription$ Exchange control of two target creatures controlled by different players.
+SVar:DBDraw:DB$ Draw | NumCards$ 3 | ConditionDefined$ Remembered | ConditionPresent$ Creature.YouCtrl | ConditionCompare$ EQ0 | SubAbility$ DBCleanup | SubAbility$ DBCleanup | StackDescription$ SpellDescription | SpellDescription$ If you control neither creature, draw three cards.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/modify_memory.txt:5]: This one is actually a bit complicated rules-wise, see here:
https://www.reddit.com/r/mtgrules/comments/v9399j/modify_memory_yes_again/

> You don't get to draw cards if one target is illegal, no matter who controlled it before the spell resolved.

Comment (kevlahnota) [reply forge-gui/res/cardsfolder/upcoming/modify_memory.txt:5]: I think extend ControlExchangeEffect class resolve function to have a param like "ReplaceFailedExchange$ ReplacementEffect" or just a flag if the exchange failed do something else..

Comment (Agetian) [reply forge-gui/res/cardsfolder/upcoming/modify_memory.txt:5]: Btw, this line seems to have two references to SubAbility$ DBCleanup, probably just a duplicate?

**Review by Agetian — APPROVED** [review]

**Pull request merged**